build.sbt

```diff
@@ -29,13 +28,14 @@ val sonatypeReleaseSettings = {
       url("https://github.com/guardian/pan-domain-authentication"),
       "scm:git:git@github.com:guardian/pan-domain-authentication.git"
     )),
+    homepage := Some(url("https://github.com/guardian/pan-domain-authentication")),
     developers := List(Developer(
       id = "GuardianEdTools",
       name = "Guardian Editorial Tools",
       email = "digitalcms.dev@theguardian.com",
       url = url("https://github.com/orgs/guardian/teams/digital-cms")
     )),
-    releaseCrossBuild := true,
+    releaseCrossBuild := false,
```

Comment:
Appreciate it's in VCS, but wonder if it's worth adding a comment here – as you mention this is a bit weird!

Reply:
done; I also went mildly overboard and included a new release step which checks the user hasn't tried to invoke sbt with a cross-build instruction while releasing - which caused me some time loss while figuring out the correct incantation

Reply:
Nice. Back of my mind: moving to autorelease for these libraries would likely reduce confusion too!